### PR TITLE
Overhauled Markdown completely

### PR DESCRIPTION
**Pull request by tycrek**
<h1>

**TL;DR** [it looks like this now]

</h1>

# Summary of changes

Please let me know if you have any questions about what I did; I'm extremely comfortable with Markdown so I should be able to answer most of them.

- Improved header with centering & better styling
- Improved Contributing section by putting it at the bottom (standard practice)
   - Also overhauled [CONTRIBUTING.md] (click to see changes on my repo)
- Added proper description
- Added collabseible section for "Modloaders explained"
- Completely overhauled actual index part of the index:
   - Three main sections are divided by separators
   - Fixed tables
   - Switched all links (except in-page references) to [Markdown reference links]
   - Archive/WIP sections are now collapsed by default (click to expand)
   - Removed some really old or irrelevant mods
   - Updated website, Discord, and GitHub information for many mods
   - Improved general Markdown readability (dummy spacing on tables, newlines between reference link groups, etc.)
- Probably more that I forgot about
- Why did I spend so much time on this
- I don't get out much huh

[it looks like this now]: https://github.com/tycrek/Index
[CONTRIBUTING.md]: https://github.com/tycrek/Index/blob/main/CONTRIBUTING.md
[Markdown reference links]: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

### CODE REVIEW

**Comment by ChiquitaV2** [comment]
This looks so good thank you so much, but I do have some issues mainly just being about the versions and links columns. I think versions should just contain the versions and not any comments maybe the "older versions may use Forge" could be a footnote or something and for links I think having separate columns for source vs support is some to keep unless you have a good reason why they should be combined. Also I can merge now if you wan to be done with this and I'll change what I think should be changed. Once again thank you so much 😊.

**Comment by Vonr** [comment]
I think footnotes should be using numbers instead of just asterisks (*) and preferably hyperlinked for easier reference.

**Comment by tycrek** [comment]
@ChiquitaV2 no problem!

> I think versions should just contain the versions and not any comments maybe the "older versions may use Forge" could be a footnote or something

Yep, I can definitely put those underneath the tables, same as the other table footnotes. Unless you have another idea?

> unless you have a good reason why they should be combined

To me it looks cleaner to have a single column with potentially 1 or two lines, rather than two columns where one might be empty. Empty columns looks out of place and filling them with something like "N/A" just adds clutter. I can revert to the previous style in about 30 seconds if you want, however.

> merge now if you wan to be done with this

I'll go back and do those few changes (once you give your input on my two notes in this comment), no problem there :)

**Comment by tycrek** [comment]
> 
> 
> I think footnotes should be using numbers instead of just asterisks (*) and preferably hyperlinked for easier reference.

Good point, I'll add those in 👍

**Comment by tycrek** [comment]
@Vonr How's this? Hyperlinking made the Markdown look pretty messy and it's easy enough to find them just by scrolling to beneath the table. But the superscripted numbers definitely look better.

image

**Comment by ChiquitaV2** [comment]
@tycrek 
>Empty columns looks out of place and filling them with something like "N/A" just adds clutter. 
I agree they look bad but for readability, I think's it's better for them to be separate.

For the hyperlinking footnote stuff, I think it's best for it to be at the bottom of the page and formatted like how it is in CONTRIBUTING.md, if nothing less than for consistency I understand it makes the raw view look messy but most people will be reading the formatted one.

**Comment by Vonr** [comment]
Normally for numeral footnotes there isn't an asterisk but both is fine

**Comment by tycrek** [comment]
@ChiquitaV2 @Vonr Expand the commit notes (the 3 dots) to see specific changes. Footnotes section at the bottom definitely looks nicer 👍 

footnotes

Let me know if there's anything else I should change, otherwise I'm good for a merge if you are 😄

**Comment by ChiquitaV2** [comment]
Thank you so much 😊 
Unless @Vonr has any other thoughts I'm ready to merge.

**Comment by BobIsMyManager** [comment]
Idk about you guys but it feels kinda weird having a row called “links” when the link to the actual clients website is in the name

**Comment by Vonr** [comment]
Looks good to me!

**Comment by pleasegivesource** [comment]
I think you should have more generic footnotes, so you can reuse them. For example, you did that for footnote #1, but you didn't for footnotes 3 and 4.

**Comment by tycrek** [comment]
> 
> 
> I think you should have more generic footnotes, so you can reuse them. For example, you did that for footnote 1, but you didn't for footnotes 3 and 4.

Ahh shoot I didn't even realize that. Must've happened when I moved them all to the bottom, missed the duplicate one

**Comment by ChiquitaV2** [comment]
I can fix that real quick

**Comment by tycrek** [comment]
That would be great, thanks!

**Comment by ChiquitaV2** [comment]
ofc but could you help me with wording idk if i should do "The client is not open-source, ..." or "This client is not open-source, ..."?

**Comment by tycrek** [comment]
> 
> 
> ofc but could you help me with wording idk if i should do "The client is not open-source, ..." or "This client is not open-source, ..."?

How's this: 

"These clients are not open-source, however their developers have published the source code for various tools each uses."

**Comment by ChiquitaV2** [comment]
ohhhh yeah that's great thanks <3
